data/templates/nginx/server.tpl.conf

```diff
@@ -29,6 +29,14 @@ server {
     ssl_session_timeout 5m;
     ssl_session_cache shared:SSL:50m;
 
+    {% if security_ciphers_compatibility == "modern" %}
+    # Ciphers with modern compatibility
+    # https://mozilla.github.io/server-side-tls/ssl-config-generator/?server=nginx-1.6.2&openssl=1.0.1t&hsts=yes&profile=modern
+    # Uncomment the following to use modern ciphers, but remove compatibility with some old clients (android < 5.0, Internet Explorer < 10, ...)
```

Comment:
Small detail but I guess we can remove this comment ? Or maybe we should put a comment explaining the use of the yunohost setting ?

Reply:
This is still informative and anybody can check against mozilla tool to see it matches. What do you think ?

Reply:
Absolutely :wink:  I was more referring about the "Uncomment the following [...]" which is no more relevant since it's not meant to be commented anymore but controlled via the setting

Reply:
I updated it due to new context.